client/components/spinner-button/docs/example.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SpinnerButton from '../index';
+
+export default class SpinnerButtonExample extends Component {
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/blocks/domain-tip">Spinner Button</a>
```

Comment:
Expected copypasta, but this URL is wrong. See also: https://github.com/Automattic/wp-calypso/pull/7534/files#r76627275